go.mod

```
@@ -1,4 +1,4 @@
-module github.com/giantswarm/release-operator/v2
+module github.com/giantswarm/release-operator/v2/v2
```

Yeah @kopiczko I think there's a bug here

---

Fix it in place. I'll take a note.

```suggestion
module github.com/giantswarm/release-operator/v2
```

When did you regenerate workflows for that one?

---

It was probably https://github.com/giantswarm/release-operator/pull/216

---

The workflow also seems to think it is behind `master` now, which isn't the case